### DISCUSSION TITLE
Change Log

### DISCUSSION THREAD

**Discussion by Vader20** [philippe44/AirConnect#502] [General]
Hi there, thanks for creating this app. It's working very smoothly on my Raspberry Pi 4B for Casting. I try to update every few weeks as you post the newer versions, and I'm sure this has been asked before, but would you able to confirm what changes? Even if it's a very small blub. You put a lot of hard work into this, and would love to know the developments, even if its small adjustments and bug fixes.

**Comment by philippe44** [comment]
Hi - There is a CHANGELOG [here](https://github.com/philippe44/AirConnect/blob/master/CHANGELOG). Now, I've recently released too many versions as a result of trying to fix an ancient issue and failing miserably... but I think I got it fixed now (not released yet, I'll do a better job at testing)

**Reply by Vader20** [reply]
Excellent, thanks again! I switched to Apple from Android and your program brought back to life all my Chromcasts devices I have all over the house :).
